JavaAssignments/src/rahulV/Assignment3_Q4.java

```diff
@@ -0,0 +1,50 @@
+package rahulV;
+
+import java.util.Scanner;
+
+public class Assignment3_Q4 {
+
+	public static void main(String[] args) {
+		char operator;
+		double num1, num2, result;
+
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Enter operator: ");
+		operator = scanner.next().charAt(0);
+		System.out.println("Enter number1 and number2: ");
+		num1 = scanner.nextDouble();
+		num2 = scanner.nextDouble();
+
+		switch (operator) {
```

Comment:
You should create separate function to write this logic and that function should return calculated value.